lib/ferrum/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Ferrum
-  VERSION = "0.13"
+  VERSION = "0.13.1"
```

Comment:
do we need this still?

Reply:
Not necessary, so I've just reverted it out 👍  Good spot